### PR TITLE
Video alt text info content update

### PR DESCRIPTION
**Pull request by melissaperreault**
Updating the info text copy to align with other space. https://github.com/Shopify/dawn/pull/840#discussion_r754712262

We are removing `make it accessible` as this is misleading but we are maintaining an instructional form.

### CODE REVIEW

**Comment by melissaperreault** [comment]
Circling back on this one, I'll connect with @KimliW to see what we can do to improve the information under the Video section and Block for `Video alt text`. i.e. https://help.shopify.com/en/manual/online-store/themes/theme-structure/theme-features#video

File: locales/en.default.schema.json [hunk 1]
```diff
@@ -385,7 +385,7 @@
             },
             "description": {
               "label": "Video alt text",
-              "info": "Describe the video to make it accessible for customers using screen readers."
+              "info": "Describe the video for customers using screen readers."
```
Comment (melissaperreault) [inline locales/en.default.schema.json:388]:
```suggestion
              "info": "Describe the video for customers using screen readers. [Learn more](https://help.shopify.com/en/manual/online-store/themes/theme-structure/theme-features#video)"
```

File: locales/en.default.schema.json [hunk 2]
```diff
@@ -1970,7 +1970,7 @@
         },
         "description": {
           "label": "Video alt text",
-          "info": "Describe the video to make it accessible for customers using screen readers."
+          "info": "Describe the video for customers using screen readers."
```
Comment (melissaperreault) [inline locales/en.default.schema.json:1973]:
```suggestion
          "info": "Describe the video for customers using screen readers. [Learn more](https://help.shopify.com/en/manual/online-store/themes/theme-structure/theme-features#video)"
```

**Comment by ludoboludo** [comment]
Let me know if/when it needs a review Meli 👍

**Comment by melissaperreault** [comment]
@ludoboludo ready for another review, the PR for the doc update shipped https://github.com/Shopify/help/pull/28718, thanks! 🙏

**Comment by melissaperreault** [comment]
Oh actually, I forgot we need to wait for translations to come back. 😇

**Review by ludoboludo — APPROVED** [review]
Looks good though 👍  Ill re approve once the translations come back 👌

**Pull request merged**